IdentityCore/src/requests/broker/MSIDSSOExtensionInteractiveTokenRequest.m

```diff
@@ -111,7 +111,8 @@ - (void)executeRequestWithCompletion:(MSIDInteractiveRequestCompletionBlock)comp
         
         ASAuthorizationSingleSignOnRequest *ssoRequest = [self.ssoProvider createRequest];
         ssoRequest.requestedOperation = [operationRequest.class operation];
-        ssoRequest.authorizationOptions = [[operationRequest jsonDictionary] msidQueryItems];
+        __auto_type queryItems = [[operationRequest jsonDictionary] msidQueryItems];
+        ssoRequest.authorizationOptions = queryItems;
```

Comment:
just curious, any reason for breaking it down to two lines? :)

Reply:
Mainly added for debugging purposes. Going to remove it later.